File: jsweet-client/src/main/java/json_schema/comm/BotInfo.java
package json_schema.comm;

public class BotInfo extends BotHandshake {

	public static final String TYPE = "botInfo";

	public BotInfo() {
		super(TYPE);
	}

	public BotInfo(String type) {
		super(type);
	}

	public String getHostName() {
		return (String) $get("host");
	}

	public Integer getPort() {
		return (Integer) $get("port");
	}
}

File: robocode2-gui/src/main/kotlin/net/robocode2/gui/model/comm/BotInfo.kt
package net.robocode2.gui.model.comm

data class BotInfo(
        val name: String,
        val version: String,
        val author: String?,
        val countryCode: String?,
        val gameTypes: Set<String>,
        val programmingLanguage: String?,
        val host: String,
        val port: Int
)
